pkg/daemon/update.go

```diff
@@ -1494,6 +1499,31 @@ func (dn *Daemon) presetUnit(unit ign3types.Unit) error {
 	return nil
 }
 
+// writeMissingAuthorizedKeys /var/home/core/.ssh/authorized_keys is not created during ignition.  Create
+// from /var/home/core/.ssh/authorized_keys.d/ignition if missing
+func (dn *Daemon) writeMissingAuthorizedKeys() error {
+	authKeyPath := filepath.Join(coreUserSSHPath, "authorized_keys")
+	authKeyFragPath := filepath.Join(coreUserSSHPath, "authorized_keys.d", "ignition")
+
+	if _, err := os.Stat(authKeyPath); err != nil {
```

Comment:
Is this correct? It is valid for the authKeyPath in OCP to be missing if the user did not supply ssh keys. We do not want to create it in all cases only when it's OKD and there are keys to write.

Reply:
the authorized_keys.d/ignition file wouldn't exist in OCP correct? We could also add a check for the ignition file and then do the copy...

Reply:
That would also take care of the OKD case when no key is given at build time